src/game/common/globaldata.cpp

```diff
@@ -728,23 +728,25 @@ GlobalData::GlobalData()
     }
 
     // crc the skirmish and multiplayer scripts.
-    File *scriptfile = g_theFileSystem->Open("Data/Scripts/SkirmishScripts.scb", File::BINARY | File::READ);
-    uint8_t buffer[CRC_BUFFER_SIZE];
     int read = 0;
+    uint8_t buffer[CRC_BUFFER_SIZE];
+    File *scriptfile = g_theFileSystem->Open("Data/Scripts/SkirmishScripts.scb", File::BINARY | File::READ);
+    if (scriptfile) {
```

Comment:
!= nullptr for all, looks good otherwise
these checks were in original but we missed them somehow so good catch

Reply:
Fixed.
I catched this by running the thyme executable under Linux - I expected it to crash - however crashes due to missing files can & should be avoided.

Reply:
This file should always be there on a typical install. I wonder, is AI broken when the file is absent?

Comment this with `// #BUGFIX` if this is no original behavior.

Reply:
its original behavior, we just missed the ifs